packages/trpc/server/routers/viewer.tsx

```diff
@@ -493,7 +493,19 @@ const loggedInViewerRouter = createProtectedRouter()
         },
         unconfirmed: {
           endTime: { gte: new Date() },
-          status: { equals: BookingStatus.PENDING },
+          OR: [
+            {
+              AND: [
+                { NOT: { recurringEventId: { equals: null } } },
+                {
+                  status: { equals: BookingStatus.PENDING },
+                },
+              ],
+            },
+            {
+              status: { equals: BookingStatus.PENDING },
+            },
+          ],
```

Comment:
Behavior similar to the upcoming status but taking into consideration just PENDING bookings.